bash/fzf-bash-completion.sh

```diff
@@ -37,11 +39,11 @@ _fzf_bash_completion_flatten_subshells() {
             elif (( count > 0 )); then
                 buffer="$line$buffer"
             else
-                echo "$line$buffer"
+                printf '%s\n' "$line$buffer"
                 buffer=
             fi
         done
-        echo -n "$buffer"
+        printf "$buffer"
```

Comment:
I think this needs a `%s` in case `$buffer` contains a %

Reply:
ah yeah not sure how I missed that. fixed!